src/components/ReportActionItem/TransactionPreview/TransactionPreviewContent.tsx

```diff
@@ -0,0 +1,288 @@
+import {canUseTouchScreen} from '@libs/DeviceCapabilities';
+import {getCleanedTagName} from '@libs/PolicyUtils';
+import variables from '@styles/variables';
+import type {TransactionPreviewUIProps} from './types';
+
+function TransactionPreviewContent({
+    isDeleted,
+    isScanning,
+    isWhisper,
+    isHovered,
+    isSettled,
+    isBillSplit,
+    isApproved,
+    isSettlementOrApprovalPartial,
+    isReviewDuplicateTransactionPage,
+    shouldShowSkeleton,
+    shouldShowRBR,
+    shouldDisableOnPress,
+    shouldShowKeepButton,
+    shouldShowDescription,
+    shouldShowMerchant,
+    shouldShowCategory,
+    shouldShowTag,
+    displayAmount,
+    category,
+    showCashOrCard,
+    tag,
+    requestCurrency,
+    merchantOrDescription,
+    previewHeaderText,
+    requestAmount,
+    splitShare,
+    receiptImages,
+    sortedParticipantAvatars,
+    containerStyles,
+    walletTermsErrors,
+    pendingAction,
+    showContextMenu,
+    offlineWithFeedbackOnClose,
+    translate,
+    navigateToReviewFields,
+    onPreviewPressed,
+    RBRmessage,
+    from,
+    to,
+    isIOU,
+}: TransactionPreviewUIProps) {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+
+    const themeStyles = useMemo(
+        () => [
+            {
+                backgroundColor: theme.cardBG,
+            },
+        ],
+        [theme.cardBG],
+    );
+
+    const shouldShowCategoryOrTag = shouldShowCategory || shouldShowTag;
+
+    const childContainer = (
+        <View>
+            <OfflineWithFeedback
+                errors={walletTermsErrors}
+                onClose={() => offlineWithFeedbackOnClose}
+                errorRowStyles={[styles.mbn1]}
+                needsOffscreenAlphaCompositing
+                pendingAction={pendingAction}
+                shouldDisableStrikeThrough={isDeleted}
+                shouldDisableOpacity={isDeleted}
+            >
+                <View
+                    style={[
+                        isScanning || isWhisper ? [styles.reportPreviewBoxHoverBorder, styles.reportContainerBorderRadius] : undefined,
+                        !onPreviewPressed ? [styles.moneyRequestPreviewBox, containerStyles, themeStyles] : {},
+                    ]}
+                >
+                    {!isDeleted && (
+                        <ReportActionItemImages
+                            images={receiptImages}
+                            isHovered={isHovered || isScanning}
+                            size={1}
+                        />
+                    )}
+                    {shouldShowSkeleton ? (
+                        <MoneyRequestSkeletonView />
+                    ) : (
+                        <View style={[styles.expenseAndReportPreviewBoxBody, styles.mtn1]}>
+                            <View style={styles.gap3}>
+                                <View style={styles.expenseAndReportPreviewTextContainer}>
+                                    {isIOU && !!from && !!to && (
```

Comment:
maybe let's group this: `isIOU && !!from && !!to` into a named const? 
something like `shouldShowXXX` or similar?